exhibitor_tls_artifacts/validators.py

```diff
@@ -0,0 +1,12 @@
+import click
+from pathlib import Path
+
+
+def validate_dir_missing(ctx, param, value):
+    """ click validator to ensure that artifacts directory does not exist """
+    path = Path(value)
+
+    if path.exists():
```

Comment:
What if the `path` is a file? Would this pass validation and fail later?

Reply:
The condition will evaluate `True` if path is a regular file/directory. However, I've just realized it will pass if the path represents a broken symlink. In which case:

```
In [9]: pathlib.Path('/tmp/badlink').exists()
Out[9]: False

In [10]: pathlib.Path('/tmp/badlink')
Out[10]: PosixPath('/tmp/badlink')

In [11]: pathlib.Path('/tmp/badlink').mkdir()
---------------------------------------------------------------------------
FileExistsError                           Traceback (most recent call last)
<ipython-input-11-330ab1d09e97> in <module>
----> 1 pathlib.Path('/tmp/badlink').mkdir()

/usr/local/Cellar/python/3.7.3/Frameworks/Python.framework/Versions/3.7/lib/python3.7/pathlib.py in mkdir(self, mode, parents, exist_ok)
   1249             self._raise_closed()
   1250         try:
-> 1251             self._accessor.mkdir(self, mode)
   1252         except FileNotFoundError:
   1253             if not parents or self.parent == self:

FileExistsError: [Errno 17] File exists: '/tmp/badlink'
```

So, I will change the condition to `if path.exists() or if path.is_symlink()`

Reply:
Let's have a test for this case for sure.

Maybe you could resolve the given `value`?

Reply:
In fact, `click.Path` can take `resolve_path=True`.